### PR TITLE
added Titan to the planets

### PR DESCRIPTION
**Pull request by dlarkinc**

### CODE REVIEW

File: src/random_planet.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import random
 
-planets = ['Mercury', 'Venus', 'Earth', 'Mars', 'Jupiter', 'Saturn', 'Uranus', 'Neptune', 'Pluto', 'Moon']
+planets = ['Mercury', 'Venus', 'Earth', 'Mars', 'Jupiter', 'Saturn', 'Uranus', 'Neptune', 'Pluto', 'Moon', 'Titan']
```
Comment (larkincunningham) [inline src/random_planet.py]: Titan? Is that a planet? And what's up with the moon and pluto? Maybe think about making a change to remove all of those.

**Review by larkincunningham — CHANGES_REQUESTED** [review]
Finished my review. See comments.

File: src/random_planet.py (hunk 1 above)
Comment (dlarkinc) [reply src/random_planet.py]: Agreed. Making changes.

**Review by larkincunningham — APPROVED** [review]
Looks perfect now.

**Pull request merged**